frontend/src/components/Donation/ProgressBar.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
This still needs work, didnt have time to come up with a smart way to make this reusable across the app and couldnt find a progress bar online that matched what we were looking for